forms.py

```diff
@@ -0,0 +1,19 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, IntegerField, SelectField, DateField
+from wtforms.validators import DataRequired, Email, Length, EqualTo, Regexp
+
+
+class LoginForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    password = PasswordField('Password', validators=[DataRequired()])
+
+
+class RegisterForm(FlaskForm):
+    name = StringField('Имя', validators=[DataRequired()])
+    surname = StringField('Фамилия', validators=[DataRequired()])
+    date_birth = DateField('Дата рождения')
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Можно использовать `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```